addons/talo/apis/channels_api.gd

```diff
@@ -7,14 +7,32 @@ class_name ChannelsAPI extends TaloAPI
 
 ## Emitted when a message is received from a channel.
 signal message_received(channel: TaloChannel, player_alias: TaloPlayerAlias, message: String)
+## Emitted when a player is joined to a channel.
+signal player_joined(channel: TaloChannel, player_alias: TaloPlayerAlias)
+## Emitted when a player is left from a channel.
+signal player_left(channel: TaloChannel, player_alias: TaloPlayerAlias)
+## Emitted when a channel's ownership transferred.
+signal channel_ownership_transferred(channel: TaloChannel, new_owner_player_alias: TaloPlayerAlias)
+## Emitted when a channel is deleted.
+signal channel_deleted(channel: TaloChannel)
+
 
 func _ready():
 	await Talo.init_completed
 	Talo.socket.message_received.connect(_on_message_received)
 
 func _on_message_received(res: String, data: Dictionary) -> void:
-	if res == "v1.channels.message":
-		message_received.emit(TaloChannel.new(data.channel), TaloPlayerAlias.new(data.playerAlias), data.message)
+	match res:
+		"v1.channels.message":
+			message_received.emit(TaloChannel.new(data.channel), TaloPlayerAlias.new(data.playerAlias), data.message)
+		"v1.channels.player-joined":
+			player_joined.emit(TaloChannel.new(data.channel), TaloPlayerAlias.new(data.playerAlias))
+		"v1.channels.player-left":
+			player_left.emit(TaloChannel.new(data.channel), TaloPlayerAlias.new(data.playerAlias))
+		"v1.channels.ownership-transferred":
+			player_left.emit(TaloChannel.new(data.channel), TaloPlayerAlias.new(data.playerAlias))
```

Comment:
this should be `data.newOwner` for the player alias

Reply:
Already fix now.